gui/src/components/StoryComponents/StoryDetails.tsx

```diff
@@ -134,6 +139,39 @@ export default function StoryDetails({
         setStoryOverview(data.story.overview);
         setStoryTestCases(data.story.test_cases);
         setStoryInstructions(data.story.instructions);
+
+        if (data.story.status === 'IN_REVIEW') {
```

Comment:
dont use the hard coded text like 'IN_REVIEW' here, we have a constant for it so use that instead